classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
Your `classroom` class should have an `attr_reader: student` method to allow you access to the `student` class 😉